pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -73,6 +73,10 @@ func Convert_v1_Disk_To_api_Disk(diskDevice *v1.Disk, disk *Disk, devicePerBus m
 		Name: "qemu",
 	}
 	disk.Alias = &Alias{Name: diskDevice.Name}
+	if diskDevice.BootOrder != 0 {
```

Comment:
Hm, so the priority starts with `1`? If so, then allowing passing in 0 is a little bit confusing. I see a few options here:

 * Using a pointer, so that we can always distinguish between `0` and no value supplied. If `0` is supplied fail the validation.
 * Allow 0 also as a priority value (and add + 1 if libvirt requires it to start wit 1). With or without a pointer.